stdlib/toolchain/Compatibility50/Overrides.cpp

```diff
@@ -34,6 +35,7 @@ OverrideSection Swift50Overrides
 __attribute__((used, section("__DATA,__swift_hooks"))) = {
   .version = 0,
   .conformsToProtocol = swift50override_conformsToProtocol,
+  .conformsToSwiftProtocol = swift51override_conformsToSwiftProtocol,
```

Comment:
I think a comment here like "We use the same implementation as the 5.1 override library" would help reduce confusion and assure future readers that this isn't a typo.